src/components/element-editor/basic-details/basic-details.tsx

```diff
@@ -75,14 +84,19 @@ export const BasicDetails = () => {
                     <Controller
                         name="label"
                         control={control}
-                        defaultValue={element?.label}
+                        defaultValue={elementName}
```

Comment:
now that the fields are controlled, there is no need to set `defaultValue`

Reply:
fixed in latest push